api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -111,39 +111,18 @@ interface DbAdminTableUser {
 
     Timestamp getCreationTime();
 
-    @Deprecated // to be removed - use getDuccBypassTime
-    Timestamp getDataUseAgreementBypassTime();
-
     Timestamp getDuccBypassTime();
 
-    @Deprecated // to be removed
-    Timestamp getDataUseAgreementCompletionTime();
-
     Timestamp getComplianceTrainingBypassTime();
 
-    @Deprecated // to be removed
-    Timestamp getComplianceTrainingCompletionTime();
```

Comment:
It's not this method specifically.  It's the same thing in spirit, but a separate implementation.

These fields represent the old way we used to refer to access modules: each module has both a completion and bypass field directly in the user table: like `compliance_training_completion_time`.  These are some of the fields we will be deleting in RW-5628, because we have transitioned away from that model.... mostly.  These are remnants.

We originally built the Admin User UI and the Reporting system to reflect those old-style DB fields.  This PR is part of the transition to new-style for Admin UI.  For Reporting, we continue to support an API which follows the old model, but we fake it using the new DB fields: see `ReportingQueryServiceImpl.getUsers()`